playwright/_impl/_connection.py

```diff
@@ -260,11 +265,10 @@ def _dispatch(self, msg: ParsedMessagePayload) -> None:
                     g.switch(self._replace_guids_with_channels(params))
             else:
                 object._channel.emit(method, self._replace_guids_with_channels(params))
-        except Exception:
-            print(
-                "Error dispatching the event",
-                "".join(traceback.format_exception(*sys.exc_info())),
-            )
+        except BaseException as exc:
+            print("Event Listener Error")
```

Comment:
```suggestion
            print("Playwright event could not be dispatched:", file=sys.stderr)
```

Reply:
This message is misleading because the event was already dispatched then the exc was raised

Reply:
Fixed